omniwitness/omniwitness.go

```diff
@@ -148,9 +154,35 @@ func Main(ctx context.Context, operatorConfig OperatorConfig, p LogStatePersiste
 			})
 		}
 
-		runDistributors(ctx, httpClient, g, distLogs, bw, operatorConfig)
+		runGitHubDistributors(ctx, httpClient, g, distLogs, bw, operatorConfig)
 	} else {
-		glog.Info("No GitHub user specified; skipping deployment of distributors")
+		glog.Info("No GitHub user specified; skipping deployment of GitHub distributors")
+	}
+	if operatorConfig.RestDistributorBaseURL != "" {
+		glog.Infof("Starting RESTful distributor for %q", operatorConfig.RestDistributorBaseURL)
+		g.Go(func() error {
```

Comment:
Is it worth breaking this func out into `runRestDistributors` or something to keep the size of this outer func down?

Reply:
Sure, done.